packages/liveblocks-react/src/factory.tsx

```diff
@@ -210,23 +242,50 @@ type RoomContextBundle<
   ];
 
   /**
-   * Returns an object that lets you get information about all the the users currently connected in the room.
+   * Returns an object that lets you get information about all the the users
+   * currently connected in the room.
    *
    * @example
    * const others = useOthers();
    *
    * // Example to map all cursors in JSX
-   * {
+   * return (
    *   others.map((user) => {
-   *     if (user.presence?.cursor == null) {
+   *     if (user.presence.cursor == null) {
    *       return null;
    *     }
    *     return <Cursor key={user.connectionId} cursor={user.presence.cursor} />
    *   })
-   * }
+   * )
    */
   useOthers(): Others<TPresence, TUserMeta>;
 
+  /**
```

Comment:
I love this documentation :chefkiss: Really detailed and clean at the same time.